api/client/BUILD

```diff
@@ -20,3 +14,42 @@ api_proto_library(
         "@envoy_api//envoy/api/v2/core:base_export",
     ],
 )
+
+# TODO(oschaaf): This is a bit of a hack, we conjure the expected _base_cc_only ourselves here
+# because it doesn't get generated for us. Also see https://github.com/grpc/grpc/issues/14145
```

Comment:
Did you try https://github.com/grpc/grpc/issues/14145#issuecomment-490809961?

Reply:
Amongst a lot of things, I also tried that and failed; I remember our `cc_grpc_library` seems to expect different inputs. Maybe this dependency needs to be updated on our side.

Reply:
Good news on this one, this is fixed [here](https://github.com/envoyproxy/nighthawk/pull/88/files#diff-04dcaeb167b260b75a61e7f95b910e72) 
(The hack broke after updating the Envoy dependency, and instead the more sane way to generate the grpc stub works with that!)